internal/plugin_test.go

```diff
@@ -173,6 +173,198 @@ subjects:
 	assertEqual(t, string(output), expected)
 }
 
+func TestGeneratePolicyExisingPlacementRuleName(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	createConfigMap(t, tmpDir, "configmap.yaml")
+	p := Plugin{}
+	var err error
+	p.baseDirectory, err = filepath.EvalSymlinks(tmpDir)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+	p.PolicyDefaults.Placement.PlacementRuleName = "plrExistingName"
+	p.PolicyDefaults.Namespace = "my-policies"
+	p.PolicyDefaults.MetadataComplianceType = "musthave"
+	patch := map[string]interface{}{
```

Comment:
The patch is irrelevant for this test. Could you please remove it?

Reply:
Removed patch